packages/amplify-cli/src/init-steps/s2-initProviders.ts

```diff
@@ -15,7 +17,10 @@ export async function initProviders(context) {
   });
 
   await sequential(initializationTasks);
-
+  // Carry environment variable values
+  if (Object.keys(categories).length > 0) {
```

Comment:
seems like this should only modify the functions category in the object?

Reply:
Agreed! I updated to modify only function instead of categories.